pages/feed/index.tsx

```diff
@@ -126,13 +126,10 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
         image={ogImage}
         twitterCard="summary_large_image"
       />
-      <div
-        className="container flex flex-col gap-16 px-2 pt-12 mx-auto md:px-16 lg:justify-end md:flex-row"
-        ref={topRef}
-      >
-        <div className="flex-col flex-1 hidden gap-6 mt-12 md:flex">
+      <div className="w-full gap-[2rem] justify-center flex flex-col md:gap-6 xl:gap-16 pt-12 md:flex-row" ref={topRef}>
+        <div className="flex-col flex-1 xl:flex hidden gap-6 mt-12">
```

Comment:
```suggestion
        <div className="w-[24%] flex-col flex-none xl:flex hidden gap-6 mt-12">
```

You are missing a `width` and `flex-none` from the `Left (userCard) div`.

With `flex-none`, the goal is to size the `Left (userCard) div` according to the `width` (that you will have to set to `24%`), rendering it fully inflexible: so it neither shrinks nor grows in relation to its parent flex container.